source/release-notes.txt

```diff
@@ -10,6 +10,39 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.37.0
+----------------
+
+*Released May 25, 2023*
+
+New Features:
+
+- Dark mode has been updated with a modern theme. If the modern theme 
+  is enabled in ``Settings>Feature Preview``, Compass defaults to 
+  the modern dark mode theme
+- When Compass detects a potential error importing a CSV file. The  
+  warning banner now shows the value, type, and issue associated with the error (:issue:`COMPASS-6543`)
+- Combine array fields into one in the import CSV preview (:issue:`COMPASS-6766`)
+- Add settings to sidebar menus (:issue:`COMPASS-6796`)
+- Show password only when user focuses on input (:issue:`COMPASS-6161`)
+- Add autocomplete support for $percentile, $median ans $$USER_ROLES (:issue:`COMPASS-6780`) (:issue:`COMPASS-6781`)
```

Comment:
(Questions) 

- Is there a ticket for the "Edit connection string" item?
- What is the new global preference.
- What about editing this item to just: Toggle state of the **Edit connection string** is based on [new global preference name]